src/main/java/model/input/file/DataFile.java

```diff
@@ -311,32 +257,84 @@ public void createMetaDataValue(String name, String type) {
 		try {
 			String fileName = this.getFile().getName();
 			String metaValue = fileName.substring(0, fileName.lastIndexOf("."));
-
-			this.metaDataValue = parseDataValue(metaValue, type);
+			Class<? extends DataValue> typeClass = DataFile.getColumnType(type);
+			this.metaDataValue = parseSimpleDataValue(metaValue, typeClass);
```

Comment:
waarom simpleDataValue. Als ik die methode bekijk kan die alleen getallen of strings zijn. Waarom(of waar als het wel kan) sta je niet alle types toe?